src/content/changelog/4-59-2-2020-03-10.md

```diff
@@ -0,0 +1,32 @@
+---
+title: '4.59.2: 2020-03-10'
+date: 2020-03-10T04:00:00.000Z
+version: 4.59.2
+changelog:
+  - API
+---
+### Version Change Notes
+
+The previous API release was assigned version number 4.14.1, while this release is tagged 4.59.2. This is an intentional change to align with internal version numbering. Subsequent releases will continue following the [semver](https://semver.org/) versioning specification.
+
+### Added
+
+* You can now create an Object Storage bucket ([POST /object-storage/buckets](/api/v4/object-storage-buckets/#post)) in the Frankfurt, DE cluster region (`eu-central-1`).
+  * To view all available Object Storage cluster regions, use the List Clusters ([GET /object-storage/clusters](/api/v4/object-storage-clusters)) endpoint.
+* Added `updated` and `created` read-only fields to Domain Record objects.
+  * Use the [Domain Records collection](https://developers.linode.com/api/v4/domains-domain-id-records) to review your Domain Record objects.
+* Added the List Object Storage Buckets in Cluster ([GET /object-storage/buckets/{clusterId}](/api/v4/object-storage-buckets-cluster-id)) endpoint.
+  * This endpoint returns a list of buckets in a specific cluster that belong to the user's Account.
+* When cloning a Linode ([POST /linode/instances/{linodeId}/clone](/api/v4/linode-instances-linode-id-clone/#post)), any tags existing on the source Linode will be cloned to the target Linode.
+
+### Changed
+
+* When a Linode goes into maintenance mode its `status` field will now display `stopped`. 
+  * To view a Linode's status, use the View Linode ([GET /linode/instances/{linodeId}](/api/v4/linode-instances-linode-id)) endpoint.
+* When creating ([POST /linode/instances](/api/v4/linode-instances/#post)) or updating ([PUT /linode/instances](/api/v4/linode-instances-linode-id/#put)) a Linode, its `root_pass` field must now also meet a password strength score requirement that is calculated internally by the API.  If the strength requirement is not met, you will receive a `Password does not meet strength requirement` error.
+
+### Fixed
+
+* Previously, when creating a Linode ([POST /linode/instances](/api/v4/linode-instances/#post)) with a root password of 6 characters a \`Password must be between 6 and 128 characters\` error was returned. This bug has been fixed and the `root_pass` field will now accept a password of 6 characters.
+* A bug has been fixed that was causing some Linode disk clones to fail ([POST /linode/instances/{linodeId}/disks/{diskId}/clone](/api/v4/linode-instances-linode-id-disks-disk-id-clone/#post)).
+* When making a payment ([POST /account/payments](/api/v4/account-payments/#post)) or staging a PayPal payment ([POST /account/payments/paypal](/account-payments-paypal/#post)) the minimum accepted amount is $5 USD. Previously, there was a bug that allowed for payment amounts lower than $5 USD.
```

Comment:
This is accurate for customers with a $0 balance or customers with more than a $5 balance. Customers in between a $0 and a $5 balance may still make a payment that matches their balance. The bug that was fixed was specifically the $0 case - the code used to allow smaller payments at $0 balance but now it requires a min of $5 at $0 balance.

Reply:
Updated!